handbook/security-policies.md

```diff
@@ -15,29 +15,29 @@ Effective security is a team effort involving the participation and support of e
 | -------------- | -------------- |
 | @GuillaumeRoss | 2022-06-01     |
 
-Fleet requires all workforce members to comply with the following acceptable use requirements and procedures, such that:
+Fleet requires all workforce members to comply with the following acceptable use requirements and procedures, such as:
```

Comment:
The meaning of such that here is that this MUST happen, such as makes it sound more like a list of examples?